sound/soc/intel/common/soc-acpi-intel-mtl-match.c

```diff
@@ -84,6 +137,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_mtl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-mtl-rt711.tplg",
 	},
-	{},
+	{
+		.link_mask = BIT(0) | BIT(2),
+		.links = rt5682_link2_max98373_link0,
+		.drv_name = "sof_sdw",
+		.sof_tplg_filename = "sof-mtl-sdw-max98373-l0-rt5682-l2.tplg",
+	},
+	{}
```

Comment:
add a comma here, that will show the diff better

Reply:
Ack